src/Microsoft.Azure.WebJobs.Storage/Blob/StorageBlobClient.cs

```diff
@@ -58,6 +58,9 @@ public Task<ServiceProperties> GetServicePropertiesAsync(CancellationToken cance
             BlobListingDetails blobListingDetails, int? maxResults, BlobContinuationToken currentToken,
             BlobRequestOptions options, OperationContext operationContext, CancellationToken cancellationToken)
         {
+            // Add WebJobs to user agent to make it easier to collect usage information from Storage telemetry
+            operationContext = OperationContextHelper.AddWebJobsUserAgentToOperationContext(operationContext);
```

Comment:
>operationContext = OperationContextHelper.AddWebJobsUserAgentToOperationContext(operationContext); [](start = 12, length = 98)

Could you make this an extension method so the call was something like 'operationContext.AddWebJobsUserAgent();'?